modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/internal/upgrade/v1_2_0/ListTypeEntryUpgradeProcess.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.list.type.internal.upgrade.v1_2_0;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Selton Guedes
+ */
+public class ListTypeEntryUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		alterTableAddColumn(
```

Comment:
I think we don't need to check this because the alterTableAddColumn is checking this [here](https://github.com/liferay/liferay-portal/blob/cf28c48b8f85bb0960dc8e57db8d533acaec7b7d/portal-kernel/src/com/liferay/portal/kernel/dao/db/BaseDBProcess.java#L262).
Please let me know if I'm wrong.

Reply:
We usually also use this information to skip updating the entries, as we want to create the column and run the updates together, just once. We can discuss about it on Monday.
Please see this example [v3_6_0/ObjectFieldUpgradeProcess.java](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_6_0/ObjectFieldUpgradeProcess.java)